src/skore/utils/_show_versions.py

```diff
@@ -0,0 +1,82 @@
+"""
+Utility methods to print system info for debugging.
+
+adapted from :func:`sklearn.show_versions`
+"""
+
+import platform
+import sys
+
+
+def _get_sys_info():
+    """System information.
+
+    Returns
+    -------
+    sys_info : dict
+        system and Python version information
+
+    """
+    python = sys.version.replace("\n", " ")
+
+    blob = [
+        ("python", python),
+        ("executable", sys.executable),
+        ("machine", platform.platform()),
+    ]
+
+    return dict(blob)
+
+
+def _get_deps_info():
+    """Overview of the installed version of main dependencies.
+
+    This function does not import the modules to collect the version numbers
+    but instead relies on standard Python package metadata.
+
+    Returns
+    -------
+    deps_info: dict
+        version information on relevant Python libraries
+
+    """
+    from importlib.metadata import PackageNotFoundError, version
+
+    import tomllib
+
+    deps = ["pip", "setuptools"]
+
+    with open("pyproject.toml", "rb") as f:
```

Comment:
I suppose that this path should be computed from the `__file__` variable. 
@thomass-dev will the pyproject.toml be there in a release ?